daemons/compute/client-go/main.go

```diff
@@ -139,6 +160,23 @@ func getStatus(ctx context.Context, client pb.DataMoverClient, uid string, maxWa
 	return rsp, nil
 }
 
+func getRequestsList(ctx context.Context, client pb.DataMoverClient, workflow string, namespace string, print bool) []string {
+	var request_uids []string
+
+	rsp, err := client.List(ctx, &pb.DataMovementListRequest{
+		Workflow:  workflow,
+		Namespace: namespace,
+	})
+
+	if err == nil {
```

Comment:
Again, consider the if case the exception. Also, no need for a local variable
```go
if err != nil {
   log.Fatalf("...")
}

return rsp.GetUids()
```

Reply:
this should now look exactly like the other request/response functions.